official/transformer/v2/transformer_main.py

```diff
@@ -376,6 +376,12 @@ def _create_optimizer(self):
       opt = tf.keras.mixed_precision.experimental.LossScaleOptimizer(
           opt, loss_scale=flags_core.get_loss_scale(self.flags_obj,
                                                     default_for_fp16="dynamic"))
+    if self.flags_obj.automatic_mixed_precision:
```

Comment:
Why is this flag here if we have the graph_rewrite flag?

Reply:
yes I'm moving this to using the graph_rewrite flag as well. Marking this PR as WIP till ready. Thanks.